modules/_commands.js

```diff
@@ -28,8 +29,8 @@ module.exports = {
                 return
               }
               if (cmd.privileged) {
-                if (bot.config.irc.control !== msg.to) {
-                  if (bot.config.irc.insecure === true) {
+                if (bot.config.get('irc.control') !== msg.to) {
```

Comment:
Should this be `irc.control !== msg.to || !irc.control_enabled`?

Reply:
We check if control channels are a thing in the next line down

Reply:
I can't leave mutliline comments :cry: 

For readability and making `insecure` clearly different from the `secure` configuration.